src/utils/graphQLClient.tsx

```diff
@@ -33,12 +50,53 @@ const sessionId = uuid();
 
 export const client = new Client({
   url: CHROMATIC_API_URL,
-  exchanges: [fetchExchange], // no cacheExchange to prevent sharing data between stories
+  exchanges: [
+    // no cacheExchange to prevent sharing data between stories
+    mapExchange({
+      onResult(result) {
+        if (result.data?.viewer === null) setCurrentToken(undefined);
```

Comment:
Does this mean every query needs to include `Query.viewer`? What about mutations?

Reply:
If the query/mutation doesn't include a `viewer` it will be `undefined` which is not equal to `null`. That's what the `=== null` check is for.

Reply:
OK 👍

Reply:
Maybe a comment for slower folks like me would help.